dotnet/src/Connectors/Connectors.OpenAI/Connectors.OpenAI.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+  </PropertyGroup>
+
+  <PropertyGroup>
```

Comment:
How does this get handled with nuget? Will this assembly be part of the main nuget package? Or a separate nuget package (I'd expect to see `nuget-package.props` or similar)

Reply:
fixed